ARCs/arc-0034.md

```diff
@@ -0,0 +1,45 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
+ 
+The Foundation’s goal is to ensure that the xGov process is defined by, and executed by, the Algorand community.
+
+The xGov program will cover evaluation of proposals regarding grants to fund early stage projects and other community input contributions.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+Being able to put measures for a vote is a significant upgrade to the power of the governors. Governors with this power **MUST**:
+- Understand the needs of the community
+- Have their interests aligned with the growth of the Algorand ecosystem
+ 
+ 
+## Rationale
+In this ARC we seek community input on issues such as:
+- How to submit a proposal?
```

Comment:
How to submit: 
Foundation to create xGov dedicated GitHub repo and make xGovs admins.

1. Problems that need a solution are created as 'issues' against the GitHub. Examples may be 'There's not enough online stake participating in consensus, new wallet creation is trending down, there's not enough visibility over key network metrics. There aren't enough wallets, or blockexplorers. Starting node is too complicated. Etc.'
2. Proposals **MUST** cite at least 1 problem raised in issues to be valid. This will separate discussion between the underlying problem and the solution. It also makes sure we're solving problems.
3. Proposals **MUST** also include:
- Straight forward arguments for how this proposal will address named problems
- An estimate of resources needed (e.g. headcount, financial)
- An estimate of deliverables and timeline
- A way for the author to be reached, (e.g. twitter handle, email, discord username)

The proposal **MUST NOT** include:
- Doxxed information or an appeal to authority (e.g. Trust me, I'm VP of Engineering at Google)
- Arguments for why the proposer themselves is the only one capable of solving the problem (e.g. We here at Zeus industries are uniquely positioned to solve this due to x)
- Any offer of compensation or bribery for accepting the proposal (e.g. Accept this and I'll give each xGov $1m)

Reply:
The proposal also **MUST NOT** include:
- obscene language or images
- any external links or downloads
- any instructions to go elsewhere on the web for more information, or for the xGovs to download anything